### ISSUE TITLE
Where are the Erlang communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/erlang#245]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Erlang supported by one or more large organizations?
* Does Erlang have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Erlang? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Erlang? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Erlang? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Erlang?
* Are there popular newsletters for Erlang?
* Is Erlang taught at programming bootcamps? (If so, what are some examples?)
* Is Erlang taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Erlang?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by juhlig** [comment]
> Is Erlang supported by one or more large organizations?

There is [Erlang Solutions](https://www.erlang-solutions.com) ("... completely devoted to the Erlang programming language...").

> Are there popular conferences for Erlang? (If so, what are some examples?)

Erlang Solutions holds and (co-)sponsors conferences around the world, for example the [Code Sync](https://codesync.global) group of conferences (Code Beam, Code Beam Lite, Code Mesh), and [Lambda Days](http://www.lambdadays.org). Most of them include Elixir talks, too. Lambda Days, however, adresses functional programming languages in general (Haskell, Scala, F#, ...), not just or even primarily Erlang (my impression was that the primary audience is Haskellers).

> Do you know of specific communities (online or offline) that are enthusiastic about Erlang? (Chat 
communities, forums, meetups, student clubs, etc)

There is the #erlang IRC channel on freenode. Though it is a rather passive channel if you're just lurking, questions are usually answered quickly and precisely. Many seasoned professional Erlangers can be found there.

Aside from that, there is the Erlang Mailing List.

> Are there specific groups or programs dedicated to mentoring people in Erlang?
> Is Erlang taught at programming bootcamps? (If so, what are some examples?)

Not sure about "mentoring" and "bootcamps" in the strict sense, but many conferences held by Erlang Solutions are accompanied by training courses.

Lambda Days often has accompanying courses offered in tandem with the local University of Krakow, Poland, but those may be about any functional programming language, not necessarily including Erlang at all.

**Comment by kytrinyx** [comment]
@juhlig Thanks so much for this, it's incredibly helpful!

I'm going to go ahead and close the issue, as I think we've got what we need 🌸
